instaloader/structures.py

```diff
@@ -11,11 +11,12 @@
 from .instaloadercontext import InstaloaderContext
 from .nodeiterator import FrozenNodeIterator, NodeIterator
 
-PostSidecarNode = namedtuple('PostSidecarNode', ['is_video', 'display_url', 'video_url'])
+PostSidecarNode = namedtuple('PostSidecarNode', ['is_video', 'display_url', 'video_url','is_last'])
```

Comment:
I tought that it was cleaner to add a property to the PostSidecar structure, instead of figuring out inside the `download_post()` function when the last one is reached.